OWNERS_ALIASES

```diff
@@ -87,3 +87,13 @@ aliases:
   #
   sig-storage-approvers: {}
   sig-storage-reviewers: {}
+
+  #
+  # SMEs
+  #
+  sme-live-migration: {}
+  sme-api-and-controllers: {}
+  sme-scheduling-and-node-plumbing: {}
+  sme-scale-and-observability: {}
+  sme-ci-and-test-framework: {}
```

Comment:
We have a sig-test already. I think I would just introduce the ci, maybe even as sig.

Reply:
Yes, we have sig-test already.
However, we need to reflect that it's two groups: people who own and create tests (usually sigs actually), and people who provide the framework and infra to run them.

Agree that it's not well defined, and something we should clean up, therefore I'm removing this line for now.